src/frontend/src/binder/select.rs

```diff
@@ -374,6 +381,72 @@ impl Binder {
         })
     }
 
+    pub fn bind_get_object_size_select(
+        &mut self,
+        output_name: &str,
+        input: &ExprImpl,
+    ) -> Result<BoundSelect> {
+        let arg = input.as_literal().ok_or_else(|| {
+            ErrorCode::BindError(format!(
+                "{output_name} only supports varchar or int literals as arguments"
+            ))
+        })?;
+        let table_object_id = self.bind_object_id(arg)?;
```

Comment:
Should I use bind the table whose size is being looked up? I did this because using `bind_table` was the existing on `Binder` that would provide the `TableId` needed to query `rw_table_stats`. But binding the table from the `pg_table_size` argument strikes me as less than safe and, instead, I should use a function that looks up the Table information but does _not_ bind the table to the query compiler context.